pkgs.nix

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+{ sources ? import ./nix/sources.nix, pkgs ? import sources.nixpkgs { } }:
+let
+  ocaml-overlay = import ./ocaml-overlay.nix { inherit sources pkgs; };
+  static-overlay = import ./static-overlay.nix;
+  normal = import sources.nixpkgs { overlays = [ ocaml-overlay ]; };
```

Comment:
Why do you extend nixpkgs with `ocaml-overlay` here? It looks like it does not affect anything

Reply:
This is for easier testing of your changes, I guess. You can do `nix build -f pkgs.nix ocamlPackages.tezos-client` and you will have to build a lot less stuff.

TBH, I initially thought we need a dynamic version of all the executables, but we don't. So maybe this is not needed indeed.

Reply:
Ok, keeping possibility to build dynamic executables is good, even if we don't actually use it right now

Reply:
Restructured `pkgs.nix` to make it extra clear what's going on.